transaction-flow.md

```diff
@@ -200,6 +202,18 @@ If a sender has been assigned an `external_id`, this value can be used to find s
   * If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you should send us an empty list of documents: `"documents": [ ]` in the request. All of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
   * If when creating senders or transactions you get the following error in the response: `"errors":{"documents":[{"error":"blank"}]}` it means that KYC requirements are not yet waived for your account. If we already approved your KYC process and so they should be, then please contact us so we can rectify the issue and update your account accordingly.
   * In case you don't do KYC on your site, then you will need to send us documents that we can use to verify the sender's identity, for more details on this please see the [API reference documentation](https://api.transferzero.com/documentation#documents).
+* Identification number: Identification number of document used with a 4-character minimum length.
```

Comment:
Don't think we need to specify the character validation. If it's less than 4 characters it's already wrong anyway.
```suggestion
* Identification number: Identification number of the document used.
```

Reply:
As per Slack messages the length of the identification number will be left in.